activiti-cloud-services-org/activiti-cloud-services-org-rest/src/test/resources/process-extensions/invalid-string-variable-extensions.json

```diff
@@ -0,0 +1,45 @@
+{
+  "id": "process-db995012-b417-4cea-a981",
+  "extensions": {
+    "mappings": {
+      "ServiceTask_06crg3b": {
+        "inputs": {
+          "78ff4945-a09e-4d25-8b91-a9b97e0b2d92": {
+            "type": "variable",
+            "value": "db995012-b417-4cea-a981-cef287de8e4a"
+          },
+          "aa4a9329-71be-4889-8265-398b68536d4a": {
+            "type": "variable",
+            "value": "ee9eb6d4-8cd6-4098-8340-13ce2f470faa"
+          },
+          "my-custom-variable": {
+            "type": "static_value",
+            "value": "ee9eb6d4-8cd6-4098-8340-13ce2f470faa"
+          }
+        },
+        "outputs": {
+          "7d74c83b-6d52-48db-a816-1d4992a40947": {
+            "type": "value",
+            "value": "test"
+          }
+        }
+      }
+    },
+    "properties": {
+      "c297ec88-0ecf-4841-9b0f-2ae814957c68": {
+        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c68",
+        "name": "stringVariable",
+        "type": "string",
+        "required": false,
+        "value": "1993-06-03"
```

Comment:
I think "1993-06-03" and "" can be real values for a process variable, so IMO the regex for string variables is too restrictive.

Reply:
Restrictions on strings have been removed 👍